openfisca_us/variables/hhs/tanf.py

```diff
@@ -74,6 +74,24 @@ class tanf_countable_income(Variable):
     documentation = "Countable income for calculating Temporary Assistance for Needy Families benefit."
     unit = "currency-USD"
 
+    def formula(spm_unit, period, parameters):
+        tanf_gross_income = spm_unit("tanf_total_gross_income", period)
+        state = spm_unit.household("state_code_str", period)
+        earned_income_deduction = parameters(
+            period
+        ).hhs.tanf.earned_income_deduction
+        return tanf_gross_income * (1 - earned_income_deduction[state])
+
+
+class tanf_total_gross_income(Variable):
+    # will include both gross earned and unearned income: read https://www.dhs.state.il.us/page.aspx?item=15814
```

Comment:
And move below - including gross + unearned is in the documentation
```suggestion
    reference: "https://www.dhs.state.il.us/page.aspx?item=15814"
```